src/couch/rebar.config.script

```diff
@@ -39,6 +39,13 @@ Version = case os:getenv("COUCHDB_VERSION") of
         string:strip(Version0, right)
 end,
 
+VersionSuffix = case os:getenv("COUCHDB_VERSION_SUFFIX") of
```

Comment:
I don't think this will t work when building from a release tarball, when we're not inside of git. Test case should be:

```
./configure -c
make dist
cd apache-couchdb-*
./configure -c
make couch
```

which I expect to fail.

Reply:
You’re right. lemme fix